src/config/passport.conf.js

```diff
@@ -8,6 +8,8 @@ import User from '../app/models/user.model.js';
 import md5 from 'md5';
 import randomstring from 'randomstring';
 
+const reservedUsernames = ['yuttadhammo', 'doug', 'martin', 'robin', 'sebastian', 'it'];
```

Comment:
good idea! the username `it` can't be used with the current validation (username must be longer than 3 chars), though.